supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -26,7 +26,7 @@ Red Hat Enterprise Linux 8 delivers a stable, secure, and consistent foundation
 
 Avoid contractions in product documentation to leave no ambiguity and to make it easier for translation and international audiences.
 
-If you are writing quickstart or other content that uses a more informal xref:#conversational-style[conversational style] (_fairly conversational_ or _more conversational_), you may use contractions. In this case, follow the guidance in _The IBM Style Guide_ on using contractions.
+If you are writing quick start or other content that uses a more informal xref:#conversational-style[conversational style] (_fairly conversational_ or _more conversational_), you may use contractions. In this case, follow the guidance in _The IBM Style Guide_ on using contractions.
```

Comment:
s/may/can

Avoid using 'may", to avoid ambiguity. Use "might" or "can". Unless you mean the month before June.

If you mean "Is permissible", then say that.

Reply:
@stoobie My update here is just changing "quickstart" to "quick start", not touching anything else in the style entry.

Additionally, this use of "may" was discussed in the original PR and it was determined to be an appropriate use of "may" per the ISG because it does mean permission.